communications/shortlinks.csv

```diff
@@ -0,0 +1,12 @@
+keyword,destination_url
+shortlinks,https://github.com/tomeshnet/toronto-community-network/blob/master/communications/shortlinks.csv
+cloud,https://cloud.tomesh.net
+github,https://github.com/tomeshnet
+gh,https://link.tomesh.net/tomeshnet
```

Comment:
```suggestion
gh,https://github.com/tomeshnet
```
Is it too hard to just keep these in sync?

Reply:
~*shrug*~ 

Just takes 1 person to forget to update.